packages/core/src/core.ts

```diff
@@ -97,7 +97,11 @@ export function exportVariable(name: string, val: any): void {
  * @param secret value of the secret
  */
 export function setSecret(secret: string): void {
-  issueCommand('add-mask', {}, secret)
+  secret.split(/\r?\n|\r/).forEach(part => {
```

Comment:
Why not just `/[\r\n]/` ?

Reply:
On Windows `os.EOL` is `\r\n`, I would argue against `\r` on its own if anything, is that realistically being used in multi line text? 🤔

Reply:
The way I read your RegEx is:

`/\r?\n|\r/`
1. any `\n`
2. any `\r\n`
3. any `\r`

Given (1) and (3), I think (2) is effectively moot.

So, it appears to me that you're RegEx is functionally equivalent to `[\r\n]+`.

Reply:
Fair, and it's a lot simpler. Changing now, thanks!